scarb/src/ops/subcommands.rs

```diff
@@ -24,6 +24,13 @@ pub fn execute_external_subcommand(cmd: &str, args: &[&OsStr], config: &Config)
     cmd.args(args);
     cmd.env(SCARB_ENV, config.app_exe()?);
     cmd.env("PATH", config.dirs().path_env());
+    cmd.env("SCARB_CACHE", config.dirs().cache_dir.path_unchecked());
+    cmd.env("SCARB_CONFIG", config.dirs().config_dir.path_unchecked());
+    cmd.env("SCARB_MANIFEST_DIR", config.root());
+    cmd.env(
+        "SCARB_LOG",
+        config.ui().verbosity().env_filter().to_string(),
```

Comment:
bug: this doesn't work

```
SCARB_LOG=scarb=trace,salsa=info scarb -vvvv xyz
```

The value of `SCARB_LOG` should be passed verbatim, but UI verbosity should still be sent somehow. I think it would be best to introduce `SCARB_UI_VERBOSITY` environment variable and pass it separately.

1. Serialization of verbosity: implement `FromStr for Verbosity` and `fmt::Display for Verbosity`
2. Add `Verbosity::from_env_var(env_var_name: &str)` and read the environment variable there. Be analogous to `EnvFilter` API.
3. I would store `SCARB_LOG` value as a `String` field in `Config`. Logs != UI